### PR TITLE
qa/tasks/mgr/test_orchestrator_cli: fix "mon update" test

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
```
2019-12-26T07:50:00.947+0000 7f6a4c94f700  0 log_channel(audit) log [DBG] : from='client.4946 -' entity='client.admin' cmd=[{"prefix": "orchestrator mon update", "num": 3, "targe
t": ["mon-mgr", ""]}]: dispatch
2019-12-26T07:50:00.947+0000 7f6a4c94f700 10 mgr.server _handle_command passing through 3
2019-12-26T07:50:00.947+0000 7f6a4a14d700 20 mgr Gil Switched to new thread state 0x55a1dfcf7200
2019-12-26T07:50:00.947+0000 7f6a4a14d700 20 mgr ~Gil Destroying new thread state 0x55a1dfcf7200
2019-12-26T07:50:00.947+0000 7f6a4a14d700 -1 mgr.server reply reply (2) No such file or directory Mons need a host spec. (host, network, name(opt))
2019-12-26T07:50:00.947+0000 7f6a4a14d700  1 -- [v2:172.21.15.38:6808/31678,v1:172.21.15.38:6809/31678] --> 172.21.15.38:0/528635034 -- mgr_command_reply(tid 0: -2 Mons need a host spec. (host, network, name(opt))) v1 -- 0x55a1dfa54780 con 0x55a1dfd5e000
```

this should address the failure of http://pulpito.ceph.com/kchai-2019-12-26_05:49:41-rados-wip-kefu-testing-2019-12-26-1055-distro-basic-smithi/4631736/

**Comment by tchaikov** [comment]
jenkins test make check arm64

File: qa/tasks/mgr/test_orchestrator_cli.py [hunk 1]
```diff
@@ -155,7 +155,6 @@ def test_host_rm(self):
         self._orch_cmd("host", "rm", "hostname")
 
     def test_mon_update(self):
-        self._orch_cmd("mon", "update", "3")
```
Comment (sebastian-philipp) [inline qa/tasks/mgr/test_orchestrator_cli.py]: interesting. this must work for rook. That's a bug in the orchestrator_cli. The code in _update_mons must not require and placement specs.

```suggestion
        # TODO: This test needs to be fixed:
        # self._orch_cmd("mon", "update", "3")
```

Comment (tchaikov) [reply qa/tasks/mgr/test_orchestrator_cli.py]: @sebastian-philipp could you file a tracker ticket on this? i think a TODO line is probably not the best way to track the pending tasks.

Comment (sebastian-philipp) [reply qa/tasks/mgr/test_orchestrator_cli.py]: > @sebastian-philipp could you file a tracker ticket on this? i think a TODO line is probably not the best way to track the pending tasks.

#32499

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2020-01-07_01:48:07-rados-wip-kefu-testing-2020-01-07-0032-distro-basic-smithi/

failures fixed / tracked by

- https://github.com/ceph/ceph/pull/32517
- https://github.com/ceph/ceph/pull/32518
- https://github.com/ceph/ceph/pull/32519
- https://tracker.ceph.com/issues/43431
- https://tracker.ceph.com/issues/43492
- https://tracker.ceph.com/issues/43494

**Pull request merged**